src/Symfony/Component/Routing/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * deprecated `ServiceRouterLoader` in favor of `ContainerLoader`
+ * deprecated `ObjectRouteLoader` in favor of `ObjectLoader`
```

Comment:
UPGRADE entries are missing for those. 
Adding one CHANGELOG+UPGRADE entry for the deprecated service in FrameworkBundle would be good as well